### PR TITLE
 [rtvs] Add new extractor

### PR DESCRIPTION
**Pull request by r0b0**
### Before submitting a *pull request* make sure you have:
- [X] At least skimmed through [adding new extractor tutorial](https://github.com/rg3/youtube-dl#adding-support-for-a-new-site) and [youtube-dl coding conventions](https://github.com/rg3/youtube-dl#youtube-dl-coding-conventions) sections
- [X] [Searched](https://github.com/rg3/youtube-dl/search?q=is%3Apr&type=Issues) the bugtracker for similar pull requests
- [X] Checked the code with [flake8](https://pypi.python.org/pypi/flake8)

### In order to be accepted and merged into youtube-dl each piece of code must be in public domain or released under [Unlicense](http://unlicense.org/). Check one of the following options:
- [X] I am the original author of this code and I am willing to release it under [Unlicense](http://unlicense.org/)
- [ ] I am not the original author of this code but it is in public domain or released under [Unlicense](http://unlicense.org/) (provide reliable evidence)

### What is the purpose of your *pull request*?
- [ ] Bug fix
- [ ] Improvement
- [X] New extractor
- [ ] New feature

---

### Description of your *pull request* and other information

Please merge extractor for Radio and Television Slovakia archive at http://www.rtvs.sk/televizia/archiv

### CODE REVIEW

File: youtube_dl/extractor/rtvs.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+
+
+class RtvsExtractorIE(InfoExtractor):
```
Comment (dstftw) [inline youtube_dl/extractor/rtvs.py]: Remove `Extractor`.

File: youtube_dl/extractor/rtvs.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+            'description': 'Galavečer pri príležitosti Medzinárodného dňa Rómov.'
+        }
+    }]
+
+    def _real_extract(self, url):
+        video_id = self._match_id(url)
+        webpage = self._download_webpage(url, video_id)
+
+        playlist_url = self._search_regex(r'"playlist": "(https?:.*)&', webpage, 'playlist_url')
+        self.to_screen("%s: Playlist URL: %s" % (video_id, playlist_url))
```
Comment (dstftw) [inline youtube_dl/extractor/rtvs.py]: Remove.

File: youtube_dl/extractor/rtvs.py [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+            'title': 'Amaro Džives - Náš deň',
+            'description': 'Galavečer pri príležitosti Medzinárodného dňa Rómov.'
+        }
+    }]
+
+    def _real_extract(self, url):
+        video_id = self._match_id(url)
+        webpage = self._download_webpage(url, video_id)
+
+        playlist_url = self._search_regex(r'"playlist": "(https?:.*)&', webpage, 'playlist_url')
```
Comment (dstftw) [inline youtube_dl/extractor/rtvs.py]: Should capture behind quotes. Non greedy.

File: youtube_dl/extractor/rtvs.py [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+        self.to_screen("%s: Playlist URL: %s" % (video_id, playlist_url))
+        playlist = self._download_json(playlist_url, video_id, "Downloading playlist")
```
Comment (dstftw) [inline youtube_dl/extractor/rtvs.py]: Single quotes.

File: youtube_dl/extractor/rtvs.py (hunk 1 above)
Comment (r0b0) [reply youtube_dl/extractor/rtvs.py]: Fixed.

File: youtube_dl/extractor/rtvs.py (hunk 2 above)
Comment (r0b0) [reply youtube_dl/extractor/rtvs.py]: Fixed.

File: youtube_dl/extractor/rtvs.py (hunk 4 above)
Comment (r0b0) [reply youtube_dl/extractor/rtvs.py]: Fixed.

File: youtube_dl/extractor/rtvs.py (hunk 3 above)
Comment (r0b0) [reply youtube_dl/extractor/rtvs.py]: Can you elaborate or post the changed regex, please? 

For example the line is:
    "playlist": "http://www.rtvs.sk/json/archive5.json?id=63118&ad=1&" + ruurl + playlistLow,
I need to capture up until the last ampersand (excluding)

**Pull request closed without merging**